scripts/3.3.0/vagrant_install_simple/script.sh

```diff
@@ -71,8 +72,16 @@ cfy bootstrap -v -p $BLUEPRINT_FILE  -i $INPUTS_FILE --install-plugins --keep-up
 #sleep 10 # wait for ui to start
 #echo finished installing cloudify-ui
 
-
+# cfy blueprints publish-archive -l https://github.com/cloudify-cosmo/cloudify-nodecellar-example/archive/master.tar.gz -b nodecellar1 -n simple-blueprint.yaml
 
 if [ "$INSTALL_SYSTEM_TESTS_REQ" = "true" ]; then
+
+    echo "host_ip: '10.10.1.10'
```

Comment:
please remove this. we don't need default values. and they are only relevant for virtualbox scenario. 

I am aware the deployment is called "installed_deployment".. correct if I am wrong, currently there's no need for the deployment to actually be installed in order for tests to pass, right ? 

We can actually install it per environment later.